src/utils/testing.tsx

```diff
@@ -96,7 +96,7 @@ export function renderStyles(node: React.ReactElement): any {
   return getComputedStyles(className)
 }
 
-export function getComputedStyles(className: string): Record<string, string> {
+export function getComputedStyles(className: string): Record<string, string | Record<string, string>> {
```

Comment:
This feels weird, but was required to make L80 of `src/utils/test-matchers.tsx` play nice. My understanding of `Record<string, string>` is that is types and object that maps strings to strings - but it seems that sometimes this will map strings to object groupings that are themselves `Record<string, string>`, as in the case of the media query key invoked on L80.

Though, I might be misunderstanding this 😅